cashier/src/main/java/com/getkeepsafe/cashier/VendorMissingException.java

```diff
@@ -1,32 +1,33 @@
-/*
- *  Copyright 2017 Keepsafe Software, Inc.
- *
- *  Licensed under the Apache License, Version 2.0 (the "License");
- *  you may not use this file except in compliance with the License.
- *  You may obtain a copy of the License at
- *
- *  http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- *  distributed under the License is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  See the License for the specific language governing permissions and
- *  limitations under the License.
- */
-
-package com.getkeepsafe.cashier;
-
-import android.support.annotation.Nullable;
-
-public class VendorMissingException extends RuntimeException {
-  public final String vendorId;
-
-  public VendorMissingException(String vendorId) {
-    this(vendorId, null);
-  }
-
-  public VendorMissingException(String vendorId, @Nullable String message) {
-    super(message);
-    this.vendorId = vendorId;
-  }
-}
+/*
```

Comment:
I think this file's content was not changed but the formatting did (e.g. spacing). Please fix the formatting.

Reply:
The only change was package for `Nullable` annotation, the rest is re-formatting (i suppose)